packages/serverpod/lib/src/models/authentication/revoked_authentication_user.spy.yaml

```diff
@@ -0,0 +1,3 @@
+### Message sent when all authentication for a user is revoked.
+class: RevokedAuthenticationUser
+serverOnly: true
```

Comment:
Question: I get that the channel it self has the userId but does it not make sense to include the userId here as well for brevity?

Reply:
We discussed it a bit, but what would it mean if the user id then is different than the one that the channel has? Should we validate that? It is basically redundant information so I opted to leave it out.